schema/logs/README.md

```diff
@@ -0,0 +1,223 @@
+# Logs Schema Support
```

Comment:
Any details on versioning support and also examples for adding custom fields.
Should the schemas be part of observability or should they be in a separate repo?

Reply:
Since other use cases are also trying to use these schemas.(Eg: Security Analytics ) should we keep these templates and schemas in a separate repo which also gives separate visibility.

Reply:
+1 on versioning support. We'll need this if the schema changes/upgrades in future.

Reply:
+1 from me as well. 

Is there a semver like document for schema versioning? Otel has their own [definition](https://opentelemetry.io/docs/reference/specification/schemas/file_format_v1.1.0/#schema-file-format-number). It's quite restrictive.